postgres-lock/src/main/resources/db/migration_lock_postgres/V1__initial_schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS locks (
+    lock_id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    locked_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
Imo, safer to have this as a Timestamp with timezone UTC (related to suggestion above)